EzMap.Api/Controllers/PoiCollectionController.cs

```diff
@@ -18,10 +18,9 @@ public class PoiCollectionController : ControllerBase
         [FromServices] IUnitOfWork uow,
         [FromServices] IIdentityService identityService)
     {
-        if (string.IsNullOrEmpty(dto.Name)
-            && string.IsNullOrEmpty(dto.Description))
+        if (!ModelState.IsValid)
         {
-            return BadRequest("Kindly fill Name, Address fields!");
```

Comment:
This message is correct, why replace with a less specific message?
And the message should come from the Validator itself, not in the controller.

Reply:
Also there should be tests for the message being return, in case you didn't have test cases for validator.